### DISCUSSION TITLE
forar to multiple ports (rustdesk)

### DISCUSSION THREAD

**Discussion by kattivius** [NginxProxyManager/nginx-proxy-manager#3811] [General]
Greetings,
I am selfhosting Rustdesk in a docker. Also nginx-proxy-manager is a stack.
The concept of setting a host and forwarding a port is clear.
Here I need to forward those ports to rustdesk.example.com:

>  

      - '21115
      - '21116
      - '21116/udp'
      - '21117'
      - '21118'
      - '21119'

I dont know how to do so.
When trying to add several hosts with same rustdesk.example.com domain, Nginx gives me an error as the domain is already among the hosts.
I think I have to use the Advanced tab but I dont know the sintax. 
Can someone help me, please?
Thank you

**Comment by jack3308** [comment]
Did you ever figure this out? I'm working through the same right now.
